common/network-common/src/main/java/org/apache/spark/network/util/JavaUtils.java

```diff
@@ -373,18 +373,22 @@ public static byte[] bufferToArray(ByteBuffer buffer) {
    * The directory will be automatically deleted when the VM shuts down.
    */
   public static File createTempDir() throws IOException {
```

Comment:
If we do not want to add this API, we can uniformly expand it into `JavaUtils.createTempDir(System.getProperty("java.io.tmpdir"), "spark")` at the place where it is called